app/src/main/java/org/wikipedia/descriptions/DescriptionEditActivity.java

```diff
@@ -67,12 +68,21 @@ public void onDescriptionEditSuccess() {
     }
 
     @Override
-    public void onPageSummaryContainerClicked(@NonNull PageTitle pageTitle) {
-        bottomSheetPresenter.show(getSupportFragmentManager(),
-                LinkPreviewDialog.newInstance(new HistoryEntry(pageTitle,
-                        getIntent().hasExtra(EXTRA_INVOKE_SOURCE) && getIntent().getSerializableExtra(EXTRA_INVOKE_SOURCE) == PAGE_ACTIVITY
-                                ? HistoryEntry.SOURCE_EDIT_DESCRIPTION : HistoryEntry.SOURCE_SUGGESTED_EDITS),
-                        null));
+    public void onBottomBarContainerClicked(@NonNull Object object) {
+        if (object instanceof PageTitle) {
```

Comment:
The `SuggestedEditsSummary` has the `PageTitle`, and if you passing `InvokeSource` in the `onBottomBarContainerClicked` would be much better than sending an `Object` value.